lms/static/sass/application.scss.mako

```diff
@@ -28,4 +28,4 @@
   @import '${env.get('THEME_NAME')}';
 % endif
 
-@import 'build-lms'; // shared app style assets/rendering
+@import 'build-lms'; // shared app style assets/rendering
```

Comment:
Hmm, I'm not exactly sure without deleting an isolated commit. Its not a huge deal in this case, but in future ones where you're locally ``git diff``ing or when viewing diffs when making the PR, this would be something to catch.